src/interfaces/contact-interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface ContactIF{
+  email: string
+  confirmEmail?: string
```

Comment:
Isn't this a required field?

Reply:
I'm not sure to be honest. The field itself isn't a value that we save/store, so when we retrieve contact info, it isn't present.
The value is only required fpr submission events for validating, so it's sort of in-between, but my thoughts were it wasn't required in the model, so perhaps it should be optional.

Reply:
OK, agreed. I thought we submitted to the BE.